src/descriptors/power_system_structs.json

```diff
@@ -9055,6 +9055,152 @@
       ],
       "supertype": "DCSource"
     },
+    {
+      "struct_name": "ZeroOrderBESS",
+      "docstring": "Parameters for the DC-side with a Battery Energy Storage System",
```

Comment:
Can you provide more detail about the model in the docstring, please. If possible a reference to where the model came from a paper or standard.